package/MDAnalysis/core/__init__.py

```diff
@@ -30,7 +30,8 @@
 :class:`~MDAnalysis.core.groups.AtomGroup` and return another
 :class:`~MDAnalysis.core.groups.AtomGroup`.
 
-:mod:`~MDAnalysis.Timeseries` are a convenient way to analyse trajectories.
+:mod:`~MDAnalysis.core.Timeseries` are a convenient way to analyse
+trajectories.
```

Comment:
Can you remove this due to deprecations already here? We should already point people to use the analysis frame work